plugins/inputs/aerospike/aerospike_test.go

```diff
@@ -156,8 +156,8 @@ func TestQuerySetsIntegration(t *testing.T) {
 	}
 
 	var acc testutil.Accumulator
-	err = acc.GatherError(a.Gather)
-	require.NoError(t, err)
+	errAs := acc.GatherError(a.Gather)
```

Comment:
Had renamed the wrong variable. Corrected it in the lastest commit.
Since aerospike-go-client v5, aerospike functions return the aerospike.Error (https://pkg.go.dev/github.com/aerospike/aerospike-client-go/v5#Error)
The accumulator on the other hand returns the Go builtin Error.
So I just renamed the variables, do you recommend some other way ?

Reply:
Ah ok, thanks for explaining.